webui/react/src/services/decoder.ts

```diff
@@ -44,6 +44,16 @@ export const mapV1Permission = (permission: Sdk.V1Permission): types.Permission
   };
 };
 
+export const mapV1UserAssignment = (
+  assignment: Sdk.V1RoleAssignmentSummary,
+): types.UserAssignment => {
+  return {
+    isGlobal: assignment.isGlobal || false,
+    roleId: assignment.roleId || 0,
+    workspaces: assignment.scopeWorkspaceIds || new Array<number>(),
```

Comment:
Just curious here why you do `new Array<number>()` instead of `[]`?

Reply:
I thought that TypeScript would care about the type of the empty array, but (in this context at least?) it doesn't need a type hint. Will do `[]` here